openhands/runtime/client/runtime.py

```diff
@@ -418,7 +418,8 @@ def run_action(self, action: Action) -> Observation:
                 response = self.session.post(
                     f'{self.api_url}/execute_action',
                     json={'action': event_to_dict(action)},
-                    timeout=action.timeout,
+                    # wait a few more seconds to get the timeout error from client side
+                    timeout=action.timeout + 5,
```

Comment:
- Is the issue runtime-dependent or happening universally across all runtimes?
- Adding "+ 5" is a bit weird: shouldn't the default timeout have a minimum value then, that is at least 5 seconds (or what it is right now +5)? Putting the change at the source is easier to maintain than adding this all over the place? 🤔

Reply:
ah actually we can do it in the send request